### PR TITLE
Update quickstart to use the Tigera operator

### PR DESCRIPTION
**Pull request by caseydavenport**
## Description

Update the quickstart guide to use the Tigera operator to install Calico. 

The operator provides lifecycle and configuration management via a minimally defined API/CRD in order to provide a better user experience.  

## Todos

- [ ] Tests
- [ ] Documentation
- [ ] Release note

## Release Note

```release-note
The quickstart guide now uses the Tigera operator to install Calico. 
```

### CODE REVIEW

**Comment by caseydavenport** [comment]
Deploy preview for *calico* ready!

Built with commit e8c824f355f96a23cd6512fc80133867f62adf21

https://deploy-preview-3622--calico.netlify.app

File: getting-started/kubernetes/quickstart.md [hunk 1]
```diff
@@ -4,40 +4,52 @@ description: Install Calico on a single-host Kubernetes cluster for testing or d
...
-   > **Note**: After installing kubeadm, do not power down or restart
-   the host. Instead, continue directly to the
-   [next section to create your cluster](#create-a-single-host-kubernetes-cluster).
-   {: .alert .alert-info}
+#### Operator based installation
+
+This quickstart guide uses the Tigera operator to install {{site.prodname}}. The operator provides lifecycle management for Calico
+exposed via the Kubernetes API defined as a custom resource definition.
+
+> **Note**: it is also possible to install Calico without an operator using Kubernetes manifests directly.
```
Comment (lmm) [inline getting-started/kubernetes/quickstart.md:37]: nit: `> **Note**: It is also...`

File: getting-started/kubernetes/quickstart.md [hunk 2]
```diff
@@ -46,81 +58,44 @@ To deploy a cluster suitable for production, refer to [Calico on Kubernetes]({{
...
-   kube-system  etcd-jbaker-1                              1/1    Running  0         6m22s
-   kube-system  kube-apiserver-jbaker-1                    1/1    Running  0         6m12s
-   kube-system  kube-controller-manager-jbaker-1           1/1    Running  0         6m16s
-   kube-system  kube-proxy-8fzp2                           1/1    Running  0         5m16s
-   kube-system  kube-scheduler-jbaker-1                    1/1    Running  0         5m41s
-   ```
-   {: .no-select-button}
-
-1. Press CTRL+C to exit `watch`.
+   > **Note**: the Tigera operator installs resources in the `calico-system` namespace. Other install methods may use
```
Comment (lmm) [inline getting-started/kubernetes/quickstart.md:94]: nit: same as the other one, casing.

File: getting-started/kubernetes/quickstart.md [hunk 3]
```diff
@@ -46,81 +58,44 @@ To deploy a cluster suitable for production, refer to [Calico on Kubernetes]({{
...
-   clusterrole.rbac.authorization.k8s.io "calico-node" created
-   clusterrolebinding.rbac.authorization.k8s.io "calico-node" created
-   daemonset.extensions "calico-node" created
-   serviceaccount "calico-node" created
-   deployment.extensions "calico-kube-controllers" created
-   serviceaccount "calico-kube-controllers" created
    ```
-   {: .no-select-button}
+   kubectl create -f {{ "/manifests/custom-resources.yaml" | absolute_url }}
+   ```
```
Comment (lmm) [inline getting-started/kubernetes/quickstart.md:84]: Could we add a quick note or link to instructions on modifying `custom-resources.yaml` if they're not using the default pod cidr?

**Comment by lxpollitt** [comment]
@caseydavenport What do you think about putting a geeky-details widget in this?

**Comment by caseydavenport** [comment]
> @caseydavenport What do you think about putting a geeky-details widget in this?

@lxpollitt yes I was thinking the same. I was wondering if we should have a "Install method" or something section added to it. WDYT?

**Comment by caseydavenport** [comment]
@lmm @lxpollitt thanks for reviewing. I think it's ready for another look!

**Pull request merged**